src/main/java/cuchaz/enigma/mapping/MappingsRenamer.java

```diff
@@ -297,4 +303,10 @@ private ClassMapping getOrCreateClassMapping(ClassEntry obfClassEntry) {
         }
         return mappingChain;
     }
+
+	public void markDirty(ClassMapping classMapping, boolean removeClass) {
```

Comment:
What will happened if I move a class to another and rename it back before saving?